src/Angor/Client/Pages/Wallet.razor

```diff
@@ -1118,9 +1123,17 @@ else
                 return;
             }
 
-            SelectRandomWordsForVerification();
+            PrepareFullVerification();
```

Comment:
This is code that is separate to the messages PR right?
The messages PR is already very big to hard to review try not add changes that are not relevant